### PR TITLE
Add Rathaus scraper

### PR DESCRIPTION
**Pull request by jdemaeyer**

### CODE REVIEW

File: rathaus/__init__.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            x.strip() for x in booth_sel.css(f'td.{td_class}::text').extract()
+            if x.strip())
+        text = text.replace('<br>', '\n')
+        assert '<' not in text, '"{text}" contains unexpected HTML tags'
+        return text
+
+    def parse_name(self, booth_sel):
+        return self._get_text(booth_sel, 'name').splitlines()[0]
+
+    def parse_description(self, booth_sel):
```
Comment (jdemaeyer) [inline rathaus/__init__.py:63]: This is pretty much just a concatenation of `goods`, maybe drop it?

File: rathaus/__init__.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        try:
+            return self._get_text(booth_sel, 'name').split('\n', 1)[1]
+        except IndexError:
+            return None
+
+    def parse_type(self, booth_sel):
+        return self.parse_tags(booth_sel)[0]
```
Comment (jdemaeyer) [inline rathaus/__init__.py:70]: If using the first tag isn't good enough we could do some manual mapping instead, the page doesn't have too many different categories

File: rathaus/__init__.py [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+            if x.lower() not in BLACKLIST
+        ]
+
+    def parse_goods(self, booth_sel):
+        BLACKLIST = ['mehr']
+        return [
+            x.strip()
+            for x in re.split(
+                # Match ',', '&', ' und', and ' u.'
+                r'(?:[,&]|\s+u(?:nd|\.))',
```
Comment (jdemaeyer) [inline rathaus/__init__.py:86]: For a few booths this will split up stuff like "Metall- und Holzfiguren" into `['Metall-', 'Holzfiguren']`

**Review by ubergesundheit — APPROVED** [review]
very good! looks nice

**Pull request merged**